Cargo.toml

```diff
@@ -249,3 +249,5 @@ codegen-units = 256 # restore default value for faster compilation
 [patch.crates-io]
 # Temporary patch until our PRs are merged.
 tar = { git = "https://github.com/qdrant/tar-rs", branch="main" }
+# Temporary patch until memmap2 0.9.5 is released.
+memmap2 = { git = "https://github.com/RazrFalcon/memmap2-rs" }
```

Comment:
We bump this to the latest release in <https://github.com/qdrant/qdrant/pull/5085>.